docs/documentation/release_notes/topics/25_0_0.adoc

```diff
@@ -245,3 +245,14 @@ on the feedback we get until the next major release, we might eventually accept
 the feature, without compromising its roadmap.
 
 For more details, see link:{adminguide_link}#_managing_organizations_[{adminguide_name}].
+
+= Support for multiple instances of a social broker in a realm
```

Comment:
Now when 25 was released, no point to add it into this file.

Reply:
True. Fixing.